msedeno23/pemexlogisticaapp
# Feat/change interfaces

**Description**
This Pull Request updates the application's interfaces by adding new styles, icons, and images to enhance the visual appearance and user experience.

**Changues**

1. > Updated the overall styling of the application interfaces.
2. > Added new icons to various parts of the application.
3. > Included new images to improve the visual representation of the application.
4. > Refactored CSS/StyleSheet files to maintain a consistent design across the application.
5. > Updated documentation to reflect the new design changes.

**Test**

1. Run the app locally.
2. Navigate through the application to ensure all interfaces display the new styles, icons, and images correctly.
3. Verify that the new icons and images are displayed appropriately in different sections of the application.
